src/extensions/replay/sessionrecording.ts

```diff
@@ -881,42 +891,31 @@ export class SessionRecording {
             this.flushBufferTimer = setTimeout(() => {
                 this._flushBuffer()
             }, RECORDING_BUFFER_TIMEOUT)
-            return this.buffer || this.clearBuffer()
+            return
```

Comment:
`this.buffer` can never be falsy anymore (and never would have been at this point anyway)